src/tagmanager/tm_parser.c

```diff
@@ -788,6 +788,64 @@ static TMParserMapGroup group_VERILOG[] = {
 	{N_("Variables"), TM_ICON_VAR, tm_tag_variable_t},
 };
 
+static TMParserMapEntry map_SYSVERILOG[] = {
```

Comment:
Yeah that can be a good idea.  But that'll require Verilog and SystemVerilog to be made consistent.  I was planning to do that in this PR once I've made up my mind about the SystemVerilog part.  It will improve Verilog ctags (more groups, more Ctrl-clickable tags), with the only drawback that the "event" type is gone (merged with "net/register", which I think is quite related to); hope that's not a problem. :)

Reply:
> For the comment we use the full kind name (3rd member) defined here

OK, I can remove the clarifications I added in parentheses.  I felt they were helpful (at least they are for me right now), and thought that the comment was merely a hint as to what that line represented so it could use the clarification ("constant" is a bit of a misnomer here, and "ifclass" seems to be an abbreviation of "interface class" which may not be obvious).

Reply:
...oh and I need to re-do the unit test for Verilog since I've changed it, of course.